.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - '4'
+  - '6'
+script:
+  - npm run coverage
+  - npm run codecov
+notifications:
+  email:
+    on_success: change
+    on_failure: change
```

Comment:
I do not have enough permission to add hipchat notification to a room. I will talk to Hajo to do that.